containers/scripts/crlite-signoff-tool.py

```diff
@@ -10,12 +11,15 @@
 import subprocess
 import sys
 import tempfile
-import workflow
 
-parser = argparse.ArgumentParser()
-parser.add_argument(
-    "--filter-bucket", help="Google Cloud Storage filter bucket name", required=True
+KINTO_RO_SERVER_URL = config(
+    "KINTO_RO_SERVER_URL", default="https://settings.stage.mozaws.net/v1/"
 )
+KINTO_BUCKET = config("KINTO_BUCKET", default="security-state-preview")
+KINTO_CRLITE_COLLECTION = config("KINTO_CRLITE_COLLECTION", default="cert-revocations")
```

Comment:
I would recommend using `REMOTE_SETTINGS_` or `RS_` but not `KINTO_`

Reply:
It would definitely be less ambiguous, but we already have many settings prefixed with "Kinto" across the project, so I guess it's not worth it to change it now. In the context of CRLite, Kinto always means Remote Settings.